### PR TITLE
Final design

### PR DESCRIPTION
**Pull request by doupalovakris**

### CODE REVIEW

File: src/static/favicon/site.webmanifest [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "name": "GHC",
```
Comment (benabraham) [inline src/static/favicon/site.webmanifest]: Mělo by být celé jméno.

File: src/templates/coffee.twig [hunk 2]
```diff
@@ -35,7 +35,7 @@
                 </h2>
               </div>
 
-              <img class="card-img-top" src="{{ coffee.img }}">
+              <img class="card-img-top" src="{{ coffee.img }}" alt=" ">
```
Comment (benabraham) [inline src/templates/coffee.twig]: prázdný string prosím, ne mezeru

File: src/templates/index.twig [hunk 3]
```diff
@@ -13,17 +13,17 @@
 
       <div class="row text-uppercase">
         <div class="col">
-          <h1 class="mb-3">Generic Hipster Coffee</h1>
-          <h2>Specialty coffee, fresh food &amp; local community.</h2>
-          <img alt="Generic hipster coffee area" class="img-fluid w-100 mt-4" src="/img/counter.jpg">
+          <h1>Generic Hipster Coffee</h1>
+          <h2>Specialty coffee,&nbsp;fresh food&nbsp;&amp;&nbsp;local community.</h2>
```
Comment (benabraham) [inline src/templates/index.twig]: &nbsp;fresh? not sure why

File: src/templates/index.twig [hunk 4]
```diff
@@ -13,17 +13,17 @@
...
         </div>
       </div>
 
-      <div class="row my-3">
+      <div class="row">
         <div class="col-12 col-sm-6 col-md-4">
           <h3 class="h4 text-uppercase">Opening hours</h3>
           <p>sunday &ndash; thursday: 8:00 &ndash; 23:00
-            <br>friday &ndash; saturday: 8:00 &ndash; 1:00</br>
+            <br>friday &ndash; saturday: 8:00 &ndash; 1:00</p>
```
Comment (benabraham) [inline src/templates/index.twig]: indenting?

**Review by benabraham — CHANGES_REQUESTED** [review]
Please make favicon readable in 16×16 size.
And indent everything correctly.

File: src/scss/_base.scss [hunk 5]
```diff
@@ -23,9 +31,35 @@ h3 {
...
+
+.hc-footer-text-white {
+    color: #ced4da;
+}
+
+.hc-footer-text-white-bigger {
+    letter-spacing: 0.08em;
+}
+
+.hc-phone-num {
```
Comment (benabraham) [inline src/scss/_base.scss]: refactor using attribute selector

File: src/scss/_base.scss [hunk 6]
```diff
@@ -23,9 +31,35 @@ h3 {
...
+
+.hc-link-underline {
+    text-decoration: underline;
+}
+
+.hc-footer-text-white {
+    color: #ced4da;
+}
+
+.hc-footer-text-white-bigger {
```
Comment (benabraham) [inline src/scss/_base.scss]: refactor using sass nesting and &

File: src/templates/menu.twig [hunk 7]
```diff
@@ -13,12 +13,12 @@
 
     <div class="container">
 
-      <div class="row mt-5 mb-3">
-        <div class="col-12 mb-4">
+      <div class="row">
+        <div class="col-12 mb-4 mb-sm-0">
           <h1>Menu</h1>
         </div>
         <div class="col-12 col-sm-6 col-md-6">
```
Comment (benabraham) [inline src/templates/menu.twig]: col-md-6 is redundant

File: src/templates/menu.twig [hunk 8]
```diff
@@ -118,23 +118,23 @@
...
+          <h2 class="hc-menu-heading">Sweets</h2>
           <p class="hc-menu-item d-flex justify-content-between"><span class="pr-3">Homemade cakes of the day</span> <span>5.0</span></p>
           <p class="hc-menu-item d-flex justify-content-between"><span class="pr-3">Key Lime and Coconut</span> <span>1.5</span></p>
           <p class="hc-menu-item d-flex justify-content-between"><span class="pr-3">Icecream</span> <span>2.0</span></p>
           <p class="hc-menu-item d-flex justify-content-between"><span class="pr-3">Tiramisu</span> <span>6.0</span></p>
         </div>
       </div>
 
-      <img alt="Generic hipster coffee menu" class="img-fluid mb-n5" src="/img/espresso-machine-detail.jpg">
+      <img alt="Generic hipster coffee menu" class="img-fluid mt-3 mb-n5" src="/img/espresso-machine-detail.jpg">
```
Comment (benabraham) [inline src/templates/menu.twig]: Alt is not correct. It's not that what we see in the image.

**Review by benabraham — CHANGES_REQUESTED** [review]
improve or fix, please

File: src/templates/index.twig (hunk 3 above)
Comment (doupalovakris) [reply src/templates/index.twig]: Nevím, co tím myslíš? - fresh food je v návrhu

Comment (benabraham) [reply src/templates/index.twig]: nbsp mezi divnejma slovama

File: src/scss/_base.scss [hunk 9]
```diff
@@ -57,15 +57,14 @@ h1, h2, h3, h4, h5, h6 {
...
-.hc-footer-text-white-bigger {
-    letter-spacing: 0.08em;
-}
+    &-bigger {
+      letter-spacing: 0.08em;
+    }
+  }
 
-.hc-phone-num {
+  a[href^="tel:+14153214567"] {
```
Comment (benabraham) [inline src/scss/_base.scss]: What if somebody uses another phone number somewhere?
A more generic one would be better.

**Review by benabraham — CHANGES_REQUESTED** [review]

**Review by benabraham — APPROVED** [review]

**Review by benabraham — APPROVED** [review]

**Pull request merged**